src/libs/KeyboardShortcut/index.js

```diff
@@ -124,6 +126,56 @@ function subscribe(key, callback, modifiers = 'shift', captureOnInputs = false)
     return () => unsubscribe(key);
 }
 
+
+/**
+ * Captures all the key inputs until explictly consumed.
+ *
+ * @param {boolean} [captureOnInputs=false] Should we capture the event on inputs too?
```

Comment:
I'm not entirely sure what this means. Could you give me an example of when we would want this to be `true`?

Reply:
I just picked it up from the old Keyboard handlers on the file but I think it better to just remove it for now. Right?